src/walinfo.c

```diff
@@ -60,6 +60,40 @@
 #define COL_WIDTH_COMBINED_SIZE   14
 #define COL_WIDTH_COMBINED_PCT    10
 
+static bool
```

Comment:
You can put all of this in configuration.c as none of configuration files are binary, so lets start with that